.github/workflows/issue-label-assign.yml

```
@@ -63,224 +63,226 @@ env:
 
  AREA_AFFIXES: >
            {
-             "prefixes":["@aws-cdk/"]
+             "prefixes":["@aws-cdk/", "aws-cdk-lib/", "aws-cdk-lib."]
```

```suggestion
             "prefixes":["@aws-cdk/", "aws-cdk-lib/", "aws-cdk-lib."]
```

Can we turn 'em in to `module/xxx` or `service/xxx` ?